packages/quantic/cypress/integration/tab/tab-selectors.ts

```diff
@@ -4,13 +4,15 @@ export const tabComponent = 'c-quantic-tab';
 
 export interface TabSelector extends ComponentSelector {
   tab: () => CypressSelector;
+  button: () => CypressSelector;
   active: () => CypressSelector;
 }
 
 export const TabSelectors: TabSelector = {
   get: () => cy.get(tabComponent),
 
   tab: () => TabSelectors.get().find('li.slds-tabs_default__item'),
+  button: () => TabSelectors.get().find('button.slds-tabs_button.slds-button'),
```

Comment:
Why is this two different things? Aren't tabs just buttons now?
Shouldn't you just update this to be like this:

```suggestion
  tab: () => TabSelectors.get().find('button.slds-tabs_button.slds-button'),
```